examples/sdk-examples/src/examples/advanced/globalFilters/GlobalFiltersExample.tsx

```diff
@@ -54,8 +54,8 @@ const Sidebar = (props: ISidebarProps) => {
                     return (
                         <SidebarItem
                             key={uri}
-                            label={title}
-                            id={uri}
+                            label={title!}
```

Comment:
We should probably handle the null case properly in SidebarItem, not just slap a ! there.

Reply:
fixed